.storybook/preview.tsx

```diff
@@ -2,12 +2,20 @@ import React, { useState } from "react";
 
 import type { Preview, StoryContext } from "@storybook/react-vite";
 
+import "../lib/reset.min.css";
 import "../lib/index.min.css";
+import "./preview.scss";
 import ColorProvider from "../src/ColorProvider";
 import type { ColorScheme } from "../src/ColorContext";
 import { MINIMAL_VIEWPORTS } from "storybook/viewport";
 import { allModes } from "./modes";
 
+const viewports = MINIMAL_VIEWPORTS;
```

Comment:
It's generally not a good idea to mutate other modules' exported values.

```suggestion
const viewports = structuredClone(MINIMAL_VIEWPORTS);
```